docs/guides/cloud-setup-invited.mdx

```diff
@@ -0,0 +1,88 @@
+
+<span id="access-credentials"></span>
+### 1. Find your access credentials
+    1. Find your API key. From the [dashboard](https://quantum.cloud.ibm.com/), create your API key, then copy it to a secure location so you can use it for authentication.
+    2. Optional: Find the instance you want to use from the [Instances](https://quantum.cloud.ibm.com/instances) page. Click the icon to copy its CRN, then save it in a secure location so you can use it to identify the instance.
+
+<span id="cloud-save"></span>
+### 2. Initialize the service (trusted environment)
+If you are working in a trusted Python environment (such as on a personal laptop or workstation), use the [`save_account()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#save_account) method to save your credentials locally, then use them to initialize the service.
+
+1. Activate a Python virtual environment, run Python in your virtual environment, and enter the following:
+
+    ```python
+    token = "<your-API-token>"
+    ```
+2. Initialize the service.
+<Admonition type="Note">
+When an instance CRN or name is passed in, only backends and jobs from that instance are available.  If an instance is not specified (allowed in Qiskit Runtime v0.40.1 and later), all backends and jobs across all instances in your account are available. In this case, when a backend is specified, an instance that can access that backend is found and used.
+</Admonition>
+
+```python
+  from qiskit_ibm_runtime import QiskitRuntimeService
+  service = QiskitRuntimeService.save_account(
+    token=token, # Your token is confidential.
+    # Do not share your key in public code.
+    instance="<IBM Cloud CRN or instance name>", # Specify the instance to use.
+    set_as_default=True, # Set these as your default credentials.
+  )
+```
+
+
+      ```python
+        from qiskit_ibm_runtime import QiskitRuntimeService
+
+    # If you named your credentials, optionally specify the name here, as follows:
+    # QiskitRuntimeService(name='account-name')
+    # If you don't specify a name, the default credentials are loaded.
+    service = QiskitRuntimeService()
+       ```
+      * If you have saved credentials for both an IBM Quantum Platform account and an IBM Cloud account, IBM Cloud is the default account used when you initialize your account, unless you specify `set_as_default=True` in your IBM Quantum Platform account when you use the `save_account()` method.
+      * If you are saving multiple accounts, consider using the `name` parameter to differentiate them.
+      * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
+      * If you don't save your credentials, you must specify them every time you start a new session.
+
+    <Admonition type="note">
+      Follow [these instructions](/docs/guides/cloud-setup-untrusted) to authenticate to IBM Cloud if you are not using a trusted environment. For example, if you are using a public comptuer.
+    </Admonition>
+
```

Comment:
should we add here also and admonition if you want to connect with the REST API instead of using Qiskit and link the users to that?